data/seeds/0002-actions.js

```diff
@@ -0,0 +1,37 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("actions")
+    .del()
```

Comment:
You forgot to install the knex-cleaner library and add it as your first seeding, in this case you are using .del() and it will delete all the existing records in your table, but I'd recommend to use .truncate() instead, since it restarts the id to 0, .del() does not restarts id, thats the only problem when we don't install the knex cleaner package, its completely optional but you should use .truncate() in the place of .del(). This is just a suggestion.